rm_vk/rm_vk/AllGroupsTableViewController/AllGroupTableViewCell.swift

```diff
@@ -29,8 +29,10 @@ final class AllGroupTableViewCell: UITableViewCell {
     // MARK: - Public methods
 
     func configureGroup(group: Group) {
```

Comment:
```suggestion
    func configure(group: Group) {
```
тут нейминг тоже group а читается как в 1 строку configure(group